src/System.ObjectModel/src/System/Collections/ObjectModel/ReadOnlyObservableCollection.cs

```diff
@@ -18,6 +18,7 @@ namespace System.Collections.ObjectModel
     [Serializable]
     [DebuggerTypeProxy(typeof(CollectionDebugView<>))]
     [DebuggerDisplay("Count = {Count}")]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public class ReadOnlyObservableCollection<T> : ReadOnlyCollection<T>, INotifyCollectionChanged, INotifyPropertyChanged
```

Comment:
In the [reference source](https://github.com/Microsoft/referencesource/blob/90b323fe52bec428fe4bd5f007e9ead6b265d553/System/compmod/system/collections/objectmodel/readonlyobservablecollection.cs#L28), this has `[TypeForwardedFrom("WindowsBase, Version=3.0.0.0, Culture=Neutral, PublicKeyToken=31bf3856ad364e35")]`. Based on @weshaggard's [comment](https://github.com/dotnet/coreclr/pull/12059#discussion_r119905431), I assume it should match here.